chapters/development_tools.adoc

```diff
@@ -27,8 +27,17 @@ Layers are optional components that augment the Vulkan system. They can intercep
   ** link:https://vulkan.lunarg.com/doc/sdk/latest/windows/debug_printf.html[Shader printf]: Debug shader code by "`printing`" any values of interest to the debug callback or stdout.
   ** link:https://vulkan.lunarg.com/doc/sdk/latest/windows/best_practices.html[Best Practices Warnings]: Highlights potential performance issues, questionable usage patterns, common mistakes.
 
-  * link:https://vulkan.lunarg.com/doc/view/latest/windows/synchronization2_layer.html[`VK_LAYER_KHRONOS_synchronization2`], the Khronos Synchronization2 layer.
-    The `VK_LAYER_KHRONOS_synchronization2` layer implements the `VK_KHR_synchronization2` extension. By default, it will disable itself if the underlying driver provides the extension.
+=== Khronos extension layers
+
+Some extensions not available everywhere can be implemented as layers. These layers can be used where the corresponding extension is not available. By default, these layers will disable themselves if the underlying driver provides the extension.
```

Comment:
Maybe it is better to just leave this  only and point to https://github.com/KhronosGroup/Vulkan-ExtensionLayer

I feel trying to list the extensions here is just setting things up to not be in sync once new layers are added

Reply:
That's a good point. I have reworded this.